app/src/main/res/drawable/ic_navigation_black_24dp.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+        android:width="24dp"
+        android:height="24dp"
+        android:viewportWidth="24.0"
+        android:viewportHeight="24.0">
+    <path
+        android:fillColor="#FF000000"
+        android:pathData="M12,2L4.5,20.29l0.71,0.71L12,18l6.79,3 0.71,-0.71z"/>
```

Comment:
What is pathData ? Bezier curve ?

Reply:
It was generated automatically by android, like every other icon in the app and I don't know what it is. I don't know either what the first 128 bytes of a png file mean, nor what is the framerate of a mp4 file by looking at the binary file. But I can still open an image and watch a movie on my computer ;).